components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryConstants.java

```diff
@@ -146,6 +146,9 @@ public class IdentityRecoveryConstants {
     public static final String SMS_OTP_GENERATE_CHAR_SET = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
     public static final String SMS_OTP_GENERATE_ALPHABET_CHAR_SET = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
     public static final String SMS_OTP_GENERATE_NUMERIC_CHAR_SET = "0123456789";
+    public static final String SMS_OTP_IS_NUMERIC = "isNumeric";
+    public static final String SMS_OTP_IS_UPPER_CASE = "isUpperCase";
+    public static final String SMS_OTP_IS_LOWER_CASE = "isLowerCase";
```

Comment:
Why do we need public constants for these?

Reply:
It has to be public since the usages are in different packages.